test/unit/utils/db/dbUrlParser_spec.js

```diff
@@ -0,0 +1,21 @@
+const { expect } = require('chai')
+
+const parseDatabaseUrl = require('../../../../src/utils/db/dbUrlParser')
+
+describe('parseDatabaseUrl', () => {
+  const dbUrl =
+    'postgres://pfforbjhkrletg:aic5oO6Cran1g3hk6mJa5QqNZB@ec2-23-21-91-97.compute-1.amazonaws.com:5432/dek11b2j1g3mfb'
+  const expected = {
+    database: 'dek11b2j1g3mfb',
+    username: 'pfforbjhkrletg',
+    password: 'aic5oO6Cran1g3hk6mJa5QqNZB',
```

Comment:
this doesn't seem secure

Reply:
it's fake test data, not real data.  It's just testing that the url format heroku uses can be parsed correctly.